src/seedu/addressbook/data/person/Email.java

```diff
@@ -38,16 +33,16 @@ public static boolean isValidEmail(String test) {
     }
 
     @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Email // instanceof handles nulls
                 && this.value.equals(((Email) other).value)); // state check
     }
+    /*
+    @Override
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.